pkg/scheduler/framework/uniquename/uniquename.go

```diff
@@ -26,7 +26,7 @@ func minInt(a, b int) int {
 	return b
 }
 
-// ClusterResourceBindingUniqueName returns a unique name for a cluster resource binding in the
+// NewlusterResourceBindingName returns a unique name for a cluster resource binding in the
```

Comment:
should be "NewClusterResourceBindingName"

missing "C" :)

Reply:
Oh, my bad. Adding it now.